core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -28,10 +28,27 @@ oppia.directive('simpleEditorBody', [
         '$scope', 'SimpleEditorManagerService',
         'explorationSaveService', 'explorationRightsService',
         'explorationWarningsService', 'QuestionIdService',
-        function($scope, SimpleEditorManagerService,
+        'StatesToQuestionsService', function($scope, SimpleEditorManagerService,
             explorationSaveService, explorationRightsService,
-            explorationWarningsService, QuestionIdService) {
+            explorationWarningsService, QuestionIdService,
+            StatesToQuestionsService) {
           $scope.data = SimpleEditorManagerService.getData();
+          $scope.chooseQuestionType = false;
+          $scope.modalWrapperVisible = false;
```

Comment:
This variable name is vague -- I'm not sure what modal wrapper we're talking about here, or even what the modal is for. Can we use a name that more clearly indicates the point of the thing that's visible?